prisma/schema.prisma

```diff
@@ -38,6 +38,9 @@ model User {
   name          String?
   emailVerified DateTime?
   image         String?
+  storageProvider String? @default("s3")
```

Comment:
I think its typo, its meant to be ipfs rit? 🤣

Reply:
One day, but now to be serious it's better for the common folks to use S3